spec/cache_dir_spec.rb

```diff
@@ -161,58 +163,58 @@
 
   describe "#download" do
     let(:url) { "https://github.com/Escape-Technologies/graphql-wordlist.git" }
-
-    before do
-      subject.instance_variable_set(:@manifest, { foo: { type: :git, url: "https://github.com/Escape-Technologies/graphql-wordlist.git", filename: "test.txt" } })
-    end
+    let(:dpath) { File.join(path, 'wordlists', 'graphql-wordlist') }
+    let(:wordlist) { Ronin::Wordlists::WordlistRepo.new("bar") }
 
     it "must download wordlist" do
-      # expect(subject).to receive(:system).with('git', 'clone', url).and_return(true)
+      expect(Ronin::Wordlists::WordlistRepo).to receive(:download).and_return(wordlist)
+
+      expect(subject.download(url)).to eq(wordlist)
     end
   end
 
   describe "#update" do
-    it "must call #each" do
-      expect(subject).to receive(:each)
+    let(:name1) { 'foo' }
+    let(:name2) { 'bar' }
+    let(:wordlist1) { Ronin::Wordlists::WordlistFile.new(File.join(subject.wordlist_dir.path,name1)) }
+    let(:wordlist2) { Ronin::Wordlists::WordlistRepo.new(File.join(subject.wordlist_dir.path,name1)) }
+    it "must iterate over all manifests and call #update" do
+      expect(subject).to receive(:[]).with(name1).and_return(wordlist1)
+      expect(subject).to receive(:[]).with(name2).and_return(wordlist2)
+      expect(wordlist1).to receive(:update)
+      expect(wordlist2).to receive(:update)
+
       subject.update
     end
   end
 
   describe "#remove" do
-    let(:wordlist) { subject["example"] }
+    let(:purge_path) { Dir.mktmpdir }
+    let(:path)       { File.join(purge_path, 'spec', 'fixtures', 'cache_dir') }
```

Comment:
Why is `spec/fixtures/cache_dir` being put into `purge_path` which is a temp dir?